documentation/SupportingDocuments/Checklist_for_API_description_review.md

```diff
@@ -0,0 +1,16 @@
+# Checklist for API description reviews
+
+
+| **Section** | **Checkpoints** |
+|---|---|
+| API Description | <ul><li>API targets and functionality is described complete and understandable</li><li>Visual illustrating API functionality and integration is included</li></ul> |
+| Use Cases | <ul><li>Use cases are described understandable</li><li>Use case name in bold, rest in normal font</li></ul> |
+| Benefits | <ul><li>Benefits are described understandable</li><li>Benefit name in bold, rest in normal font</li></ul> |
+| API Portfolio: | <ul><li>Prefix correct (see column section)</li><li>Correct portfolio is shown</li></ul> |
+| SubProject Wiki: | <ul><li>Prefix correct (see column section)</li><li>Link is correct and named with SubProject or Independent Sandbox</li><li>"(incl. how to meet the team)" is added</li></ul> |
+| API Wiki: | <ul><li>Prefix correct (see column section)</li><li>Link is correct and named with API</li></ul> |
+| API Repository: | <ul><li>Prefix correct (see column section)</li><li>Link is correct and named with API</li></ul> |
+| API Repository Status: | <ul><li>Prefix correct (see column section)</li><li>Status is correct (Sandbox, Incubating, Graduated)</li></ul> |
+| API Status: | <ul><li>Prefix correct (see column section)</li><li>Status is correct (Upcoming, In progress, Release candidate, Initial, Stable)</li></ul> |
+| API Version(s) and Release Date(s): | <ul><li>Prefix correct (see column section)</li><li>Public release API versions are listed</li><li>Format is "v0.0.0 (dd.mm.yyyy), Spring25 meta-release"</li><li>Links are correct</li></ul> |
```

Comment:
```suggestion
| API Version(s) and Release Date(s): | <ul><li>Prefix correct (see column section)</li><li>Public release API versions are listed</li><li>Format is "vx.y.z (dd.mm.yyyy), Spring25 meta-release"</li><li>Links are correct</li></ul> |
```

Reply:
"Links are correct" ... is there a definition where the link should go to? Just to the release of the API Repository (which contains the release description and with that the context and the links to the API definition) (my preferred option) or directly to the YAML (in this case extra care to be taken that it is going to the tagged version of the release).